scripts/download-openapi.ts

```diff
@@ -3,8 +3,9 @@ const { Readable } = require('stream');
 import { finished } from 'stream/promises';
 
 const downloadOpenapi = async () => {
-  const url = process.argv[2] ? new URL(process.argv[2]) : new URL('https://dev.demo.ivyteam.io');
-  await download('openapi-dev.yaml', new URL('api/openapi.yaml', url));
+  const url = process.argv[2] ? new URL(process.argv[2]) : new URL('https://neo.demo.ivyteam.io');
```

Comment:
Not sure if this will be a stable url ;)

Reply:
^^I'm aware, need to find a better solution